spec/helpers/action_view_extension_spec.rb

```diff
@@ -50,6 +50,14 @@
     it { is_expected.to include Money.default_currency.symbol }
   end
 
+  describe '#humanized_money_with_iso' do
+    subject { helper.humanized_money_with_iso Money.new(12500) }
+    it { is_expected.to be_a String }
+    it { is_expected.not_to include Money.default_currency.decimal_mark }
+    it { is_expected.to include Money.default_currency.iso_code }
+    it { is_expected.not_to include Money.default_currency.symbol }
```

Comment:
I think instead of having these ^^ 4 specs it's better to have one checking for the exact output match. Otherwise adding extra symbols to your new method will still result in this test passing.